assemblies/assembly-rhdh-telemetry.adoc

```diff
@@ -1,5 +1,5 @@
 [id="assembly-rhdh-telemetry"]
-= Telemetry data collection
+= Telemetry data collection and analysis
 
 The telemetry data collection feature helps in collecting and analyzing the telemetry data to improve your experience with {product}. This feature is enabled by default.
 
```

Comment:
Consider moving the content in the note currently at lines 6-9 to the right place: in the _Disabling telemetry_ procedure introduction.